pbs/pbsresponse.go

```diff
@@ -1,26 +1,38 @@
 package pbs
 
+// PBSBid is a bid from the auction. These are produced by Adapters, and target a particular Ad Unit.
 type PBSBid struct {
-	BidID             string            `json:"bid_id"`
-	AdUnitCode        string            `json:"code"`
-	Creative_id       string            `json:"creative_id,omitempty"`
-	BidderCode        string            `json:"bidder"`
-	BidHash           string            `json:"-"` // this is the hash of the bidder's unique bid identifier for blockchain. Should not be sent to browser.
-	Price             float64           `json:"price"`
-	Currency          string            `json:"currency,omitempty"`
-	NURL              string            `json:"nurl,omitempty"`
-	Adm               string            `json:"adm,omitempty"`
-	Width             uint64            `json:"width,omitempty"`
+	// BidID identifies the Bid Request within the Ad Unit which this Bid targets. It should match one of
+	// the values inside PBSRequest.AdUnits[i].Bids[j].BidID.
+	BidID string `json:"bid_id"`
+	// AdUnitCode identifies the AdUnit which this Bid targets.
+	// It should match one of PBSRequest.AdUnits[i].Code, where "i" matches the AdUnit used in
+	// as BidID.
+	AdUnitCode  string `json:"code"`
+	Creative_id string `json:"creative_id,omitempty"`
+	BidderCode  string `json:"bidder"`
+	// BidHash is the hash of the bidder's unique bid identifier for blockchain. It should not be sent to browser.
+	BidHash string `json:"-"`
+	// Price is the cpm, in US Dollars, which the bidder is willing to pay if this bid is chosen.
+	// TODO: Add support for other currencies someday.
+	Price float64 `json:"price"`
+	NURL  string  `json:"nurl,omitempty"`
```

Comment:
Notify URL is explained in the OpenRTB spec; it sends back a URL to call to get creatives instead of sending the markup in the Adm field. These are mutually exclusive i think.